attributes/default.rb

```diff
@@ -252,7 +252,20 @@
 end
 # rubocop:enable Metrics/BlockLength
 
+
+
+
 # auditd config
+default['os-hardening']['auditd']['log_file'] = '/var/log/audit/audit.log'
+default['os-hardening']['auditd']['log_format'] = 'RAW'
+default['os-hardening']['auditd']['max_log_file_action'] = 'keep_logs'
+default['os-hardening']['auditd']['space_left'] = 75
+default['os-hardening']['auditd']['action_mail_acct'] = 'root'
+default['os-hardening']['auditd']['space_left_action'] = 'EMAIL'
+default['os-hardening']['auditd']['admin_space_left'] = 50
+default['os-hardening']['auditd']['admin_space_left_action'] = 'halt'
```

Comment:
in the old template we had `suspend`, is there any special reason to change that to `halt`?